app/routes/circulars/submit.tsx

```diff
@@ -33,13 +33,13 @@ export async function action({ request }: DataFunctionArgs) {
 }
 
 export default function Submit(props: FormProps) {
-  const defaultSubjectValid = subjectIsValid(props.subject ?? '')
+  const defaultSubjectValid = subjectLineIsValid(props.subject ?? '')
```

Comment:
Why the name change?

Reply:
I wrote the new function, and when I implemented it in the submit page I saw the old function was defined there already but didnt do anything. I guess I forgot there was even a function there already. I can change the name though, that will keep the merge smaller